packages/frontend/package.json

```diff
@@ -26,6 +26,7 @@
         "@uidotdev/usehooks": "^2.1.0",
         "@unirep-app/circuits": "1.0.0",
         "@unirep/core": "2.0.0-beta-4",
+        "@unirep/utils": "^2.0.0-beta-5",
```

Comment:
core 2.0.0-beta-6 should be merged in main
you can just install `@unirep/core: "2.0.0-beta-6"`

Reply:
or merge the main branch